libs/wallet/src/connect-dialog/connect-dialog.tsx

```diff
@@ -251,39 +238,6 @@ const SelectedForm = ({
   reset: () => void;
   onConnect: () => void;
 }) => {
-  if (connector instanceof RestConnector) {
```

Comment:
chore (blocking): unfortunately we still need the rest connector form to support hosted wallet